WalletWasabi/Wallets/WalletFilterProcessor.cs

```diff
@@ -55,6 +60,7 @@ public WalletFilterProcessor(KeyManager keyManager, BitcoinStore bitcoinStore, T
 	/// <remarks>Guards <see cref="SynchronizationRequests"/>.</remarks>
 	private object Lock { get; } = new();
 	private SemaphoreSlim SynchronizationRequestsSemaphore { get; } = new(initialCount: 0);
+	private SemaphoreSlim PrefetchBlocksDownloadSemaphore { get; } = new(initialCount: MaxNumberOfParallelBlockDownloads - 1);
```

Comment:
`- 1` because downloading a block the standard way is prioritized therefore doesn't use the `Semaphore`